clouds/snowflake/modules/doc/h3/H3_POLYFILL.md

```diff
@@ -1,15 +1,21 @@
 ## H3_POLYFILL
 
 ```sql:signature
-H3_POLYFILL(geography, resolution)
+H3_POLYFILL(geography, resolution)  
+H3_POLYFILL(geography, resolution, mode)
 ```
 
 **Description**
 
-Returns an array with all the H3 cell indexes **with centers** contained in a given polygon. It will return `null` on error (invalid geography type or resolution out of bounds).
+Returns an array with all H3 cell indicies contained in the given polygon. There are three modes which decide if a H3 cell is contained in the polygon:  
```

Comment:
Actually maybe you are right and we should use `indicies` instead of `indexes` but would keep it as it is for now, and think of it as something to update in all the docs.